pkg/volume/local/local.go

```diff
@@ -285,7 +285,7 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 
 		if len(refs) > 0 {
 			fsGroupNew := int64(*fsGroup)
-			fsGroupSame, fsGroupOld, err := volume.IsSameFSGroup(m.globalPath, fsGroupNew)
+			fsGroupSame, fsGroupOld, err := volume.IsSameFSGroup(m.mounter.KubeletPath(m.globalPath), fsGroupNew)
```

Comment:
@jsafrane pointed out a problem with using a wrapped path in a [different PR](https://github.com/kubernetes/kubernetes/pull/61489#pullrequestreview-110032850).  So I think we need to pass in `mounter` and invoke `stat` on the host instead